graph/min_cost_b_flow/verifier.cpp

```diff
@@ -0,0 +1,29 @@
+#include "testlib.h"
+#include "params.h"
+
+int main() {
+    registerValidation();
+    const size_t n = inf.readInt(0, N_MAX, "n");
+    inf.readSpace();
+    const size_t m = inf.readInt(0, M_MAX, "m");
+    inf.readChar(LF);
+    for (size_t i = 0; i < n; ++i) {
+        inf.readLong(-U_MAX, U_MAX, "b"); // b
+        inf.readChar(LF);
+    }
+    for (size_t i = 0; i < m; ++i) {
+        inf.readInt(0, n - 1, "s");
+        inf.readSpace();
+        inf.readInt(0, n - 1, "t");
+        inf.readSpace();
+        const int64_t lower = inf.readLong(-U_MAX, U_MAX, "l");
+        inf.readSpace();
+        const int64_t upper = inf.readLong(-U_MAX, U_MAX, "u");
+        ensure(lower <= upper);
```

Comment:
This condition is not listed in the constraints section (I think it's good to have it there).

Reply:
Good catch, I'll add it.